### PR TITLE
Update enable-usage-analytics.md

### PR DESCRIPTION
**Pull request by msbemba**
The UI changed for Enabling template app

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4561

File: microsoft-365/admin/usage-analytics/enable-usage-analytics.md [hunk 1]
```diff
@@ -48,11 +48,11 @@ To enable the template app, you have to be a **Global administrator**.
   
 See [about admin roles](../add-users/about-admin-roles.md) for more information. 
   
-1. In the admin center, go to the **Reports** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=2074756" target="_blank">Usage</a> page. 
+1. In the admin center, go to the **Settings** \> **Org settings** \> **Service** Tab.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/usage-analytics/enable-usage-analytics.md]:
```suggestion
1. In the admin center, go to the **Settings** \> **Org settings** \> **Services** Tab. 
```

File: microsoft-365/admin/usage-analytics/enable-usage-analytics.md [hunk 2]
```diff
@@ -48,11 +48,11 @@ To enable the template app, you have to be a **Global administrator**.
   
 See [about admin roles](../add-users/about-admin-roles.md) for more information. 
   
-1. In the admin center, go to the **Reports** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=2074756" target="_blank">Usage</a> page. 
+1. In the admin center, go to the **Settings** \> **Org settings** \> **Service** Tab. 
     
-2. On the **Usage** page, locate the **Microsoft 365 usage analytics** card, and select **Get started**.
+2. On the **Services** tab, Select  **Reports** .
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/usage-analytics/enable-usage-analytics.md]:
```suggestion
2. On the **Services** tab, Select  **Reports**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@scotvidican Please help us in merging this PR. Thanks.

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
Gentle reminder, @kwekuako Please merge this PR. Thanks.

File: microsoft-365/admin/usage-analytics/enable-usage-analytics.md [hunk 3]
```diff
@@ -48,11 +48,11 @@ To enable the template app, you have to be a **Global administrator**.
   
 See [about admin roles](../add-users/about-admin-roles.md) for more information. 
   
-1. In the admin center, go to the **Reports** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=2074756" target="_blank">Usage</a> page. 
+1. In the admin center, go to the **Settings** \> **Org settings** \> **Services** Tab.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/usage-analytics/enable-usage-analytics.md]:
```suggestion
1. In the admin center, go to the **Settings** \> **Org settings** \> **Services** tab. 
```

File: microsoft-365/admin/usage-analytics/enable-usage-analytics.md [hunk 4]
```diff
@@ -48,11 +48,11 @@ To enable the template app, you have to be a **Global administrator**.
   
 See [about admin roles](../add-users/about-admin-roles.md) for more information. 
   
-1. In the admin center, go to the **Reports** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=2074756" target="_blank">Usage</a> page. 
+1. In the admin center, go to the **Settings** \> **Org settings** \> **Services** Tab. 
     
-2. On the **Usage** page, locate the **Microsoft 365 usage analytics** card, and select **Get started**.
+2. On the **Services** tab, Select  **Reports**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/usage-analytics/enable-usage-analytics.md]:
```suggestion
2. On the **Services** tab, select  **Reports**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @msbemba - two more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**